source/systems/MSU-HPC_user_guide.rst

```diff
@@ -94,13 +94,11 @@ resets and training to keep your account active.
 
 **Password Resets**
 
-A password reset can only be done by phone during normal business days
-(M-F) from 08:00-12:00 and 13:00-17:00 Central Time. If you forget
-your password, please call: *(662) 325-9146*. If you do not get an
-answer keep calling every 30 minutes. If busy, call again every 15
-minutes. The MSU HPC2 admin will verify your ID, unlock your account,
-and issue a temporary password that is only valid to access the TAPS_
-portal.
+If you need to reset your password, please navigate to the TAPS_ portal
+and select "Forgot your password?". Enter your username, and then select
+"Request Password Reset". If your account is locked or disabled, or the
+"Password Reset" feature isn't working, please send a ticket to the MSU
+help desk.
```

Comment:
I'd rewrite:
...if the Password Reset feature isn't working, please open a help ticket. Send email to the MSU help desk address (https://docs.rdhpcs.noaa.gov/help/index.html#getting-help) with Password Reset in the subject line.

Reply:
I made the changes in commit 4e1789f